ruby-audit-logs-example/views/send_events.erb

```diff
@@ -1,102 +1,220 @@
+                            <div>
+                                <a href="#export-events" class="remove-style"><button class="remove-style">Export Events</button></a>
+                            </div>                            
                         </div>
-                    </div>
-                    <hr style="height:0.5px;width:100%;">
-                    <form  method="POST" action="/send_event">
-                        <div class="flex margin-y-15">                                                                
+                        <div class="flex space-evenly width-11vw content-button tab">
+                            <img src="<%= url("images/eye.svg")%>" class="icon" alt="settings icon">
                             <div>
-                                <button class="box" name="event" id="user_signed_in" value="0" type="submit" onclick="showSnackbar()">                            
-                                    <div class="flex width-100p">
-                                        <p>User Signed In</p> 
-                                    </div>                           
-                                </button>                    
-                                <button class="box" name="event" id="user_logged_out" value="1" type="submit" onclick="showSnackbar()">                            
-                                    <div class="flex width-100p">
-                                        <p>User Logged Out</p> 
-                                    </div>                           
-                                </button> 
-                            </div>
+                                <a href="/events?intent=audit_logs&organization_id=<%=@organization.id%>" class="remove-style"><button class="remove-style">View Events</button></a>
+                            </div>                            
+                        </div>
+                        <div class="flex space-evenly width-11vw content-button tab">
+                            <img src="<%= url("images/share-2.svg")%>" class="icon" alt="settings icon">
                             <div>
-                                <button class="box" name="event" id="user_org_deleted" value="2" type="submit" onclick="showSnackbar()">                            
-                                    <div class="flex width-100p">
-                                        <p>User Organization Deleted</p> 
-                                    </div>                           
-                                </button>                    
-                                <button class="box" name="event" id="user_connection_deleted" value="3" type="submit" onclick="showSnackbar()">                            
+                                <a href="/events?intent=log_streams&organization_id=<%=@organization.id%>" class="remove-style"><button class="remove-style">Configure Log Streams</button></a>
```

Comment:
Getting this error when clicking the Log Streams button. Can you update the intent to be "audit_logs"?
![CleanShot 2023-03-27 at 14 09 50](https://user-images.githubusercontent.com/47123183/228055515-2afd530c-c072-4a6c-9525-35cb5ad39552.png)